executor/mpp_gather.go

```diff
@@ -70,6 +72,7 @@ func (e *MPPGather) appendMPPDispatchReq(pf *plannercore.Fragment, tasks []*kv.M
 		if err != nil {
 			return errors.Trace(err)
 		}
+		logutil.BgLogger().Info("Dispatch mpp task", zap.Uint64("timestamp", mppTask.StartTs), zap.Int64("ID", mppTask.ID), zap.String("address", mppTask.Meta.GetAddress()), zap.String("plan", plannercore.ToString(pf.ExchangeSender)))
```

Comment:
Should we remove this line before merging?

Reply:
I add this log intentionally and removed the original logs. This log is more accurate and specific.